util/topgen/c.py

```diff
@@ -116,3 +177,16 @@ def plic_targets(self):
         enum.add_last_constant("Final PLIC target")
 
         return enum
+
+    def plic_sources(self):
+        return self._plic_sources
+
+    def plic_interrupts(self):
+        return self._plic_interrupts
```

Comment:
This feels a little "Java-y". Is there a strong reason not to just expose the attributes?